src/main/java/pl/javastart/task/Room.java

```diff
@@ -0,0 +1,32 @@
+package pl.javastart.task;
+
+public class Room {
+    private final double surface;
+    private double temperature;
+    private final double minTemperature;
+    private final boolean airCondition;
+
+    public Room(double surface, double temperature, double minTemperature, boolean airCondition) {
+        this.surface = surface;
+        this.temperature = temperature;
+        this.minTemperature = minTemperature;
+        this.airCondition = airCondition;
+    }
+
+    String getInfo() {
+        String format = "powierzchnia pomieszczenia: %.2f m.kw., aktualna temperatura: %.1f\u00B0C, " +
```

Comment:
Jeżeli masz kodowanie ustawione na utf8, to teoretycznie powinno się dac zapisać po prostu "°C" bezpośrednio w kodzie, ale nie musisz się tym przejmować